tests/cases/php/global-fallbacks

```diff
@@ -0,0 +1,24 @@
+---DESCRIPTION---
+No global fallbacks
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor;
+
+date(DATE_ATOM);
+
+---FIXED---
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor;
+
+use function date;
```

Comment:
I can see this becoming annoying for many PHP functions e.g. all `strlen` and similar which gets used everywhere. Even Java auto-import [`java.lang`](https://docs.oracle.com/javase/7/docs/api/java/lang/package-summary.html) to avoid too many imports. PHP provides the fallback too for functions (but not for classes IIRC).

If anything I would forbid defining a `strlen` that is not `\strlen`, which tries to avoid the ambiguity.

Reply:
Some functions get a very-minor performance boost by not checking the fallback. But more importantly prevents https://www.giorgiosironi.com/2010/09/monkey-patching-in-php.html (as used by Symfony's [`ClockMock`](https://github.com/symfony/phpunit-bridge/blob/master/ClockMock.php)).

Might be annoying when not using an IDE though...